src/embedded_curve.rs

```diff
@@ -0,0 +1,54 @@
+use num::BigUint;
```

Comment:
Is this file obsolete now?

Reply:
I'll put the JubJub curve specification here - I was thinking about it as a place to put common curve definitions